architecture/index.md

```diff
@@ -1,30 +1,46 @@
 # AppStudio
 
 ## Overview
-AppStudio is a platform for building integrated experiences that streamline, consolidate, and secure the application lifecycle.
+AppStudio is a platform for building integrated software that streamlines, consolidates, and secures the development lifecycle.
 
 
 ### Goals
-- Compose cloud native applications that consist of multiple components and services
-- Provide managed application lifecycle
-- Rapid bootstrapping of applications
-- Fast onboarding of applications to the cloud
-- Supports both existing and new applications
-- Provide APIs to manage your application lifecycle
-- Provide a surface for partners to integrate into the application lifecycle
-
-## Architecture Goals and Constraints
-- Robust delivery automation: Establish continuous delivery practices but also deliver operational tooling.
+- Compose software that consists of multiple components, from multiple repositories.
+- Provide transparency on the software supply chain, including both what makes up the software and how it was built.
+- Provide a way for software teams to release to destinations under the control of their SRE or release engineering team(s).
+- Rapid bootstrapping of new software projects
+- Support both existing and new projects
+- Provide APIs to manage your software lifecycle
+- Provide a surface for partners to integrate, add value
+
+## Architecture Goals
+
+- Build artifacts once with enough data to determine suitability for releasing.
+- Build artifacts once that can be released to multiple locations, use cases.
+- Build artifacts should be fully determined by git, not by transient state of the build system. Employ tools like [renovate](https://docs.renovatebot.com/) to manage dependency updates.
+- "Shift left" the decisions for releasing into PRs; you should be able to release artifacts from a PR as soon as it is merged.
 - Just in time scaling: In contrast to “just in case” scaling. The system should be able to scale without capacity reserved ahead of time.
-- Static stability: the overall system continues to work when a dependency is impaired
-- Each subservice can fulfill its primary use cases independently, without relying on  other systems’ availability.
+- Static stability: the overall system continues to work when a dependency is impaired.
```

Comment:
To @twaugh 's comment, I think we can add something about "avoiding flag days" in here. It is also related to our decision to use Tekton (so lower might be an option too), we shouldn't make changes that can break builds ... instead we propose changes and provide a window for accepting them

Reply:
Added in d3bc7b3.